website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -21,14 +21,13 @@ The Admin Settings page allows [superusers](#add-super-admins) on self-hosted in
 If you don’t see the “Admin Settings” icon on the top-right corner, you’re not a super admin for the Appsmith instance and must be granted admin privileges by an existing super admin.
 :::
 
- <ZoomImage
-        src="/img/admin-settings-topbar.png"
-        alt="Admin Settings option is available in the left sidebar"
-        caption="Access admin settings from the top-right corner of the homepage."
-        lazyLoad="true"
+<ZoomImage
```

Comment:
Why is this change needed?

Reply:
sorry, my fomatter changed it when I auto saved it, I'll remove them